gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/GemFireCacheImpl.java

```diff
@@ -689,6 +690,45 @@ private void enqueueOldEntry(RegionEntry oldRe, Map<Object, BlockingQueue<Region
     }
   }
 
+  final void printOldEntries(Region region, final Object entryKey,
```

Comment:
No body is using this now. Do you intend to keep this for debugging purpose later. Or do you want to write a procedure to dump this?

Reply:
Intend to keep for debugging purpose. Writing a procedure will also be a good idea for manual debugging in case required.